### ISSUE TITLE
hxml tasks don't work if there's no hxml in the project root

### ISSUE THREAD

**Issue by back2dos** [vshaxe/vshaxe#296]
Our project has it's hxmls nested deeply and I've set the `haxe.displayConfigurations` in the `settings.json` accordingly. If I have an empty hxml in the project root, the compilation works fine with the "active configuration", but as soon as I remove the file I get:

```
The haxe task detection didn't contribute a task for the following configuration:
{
    "type": "haxe",
    "args": "active configuration",
    "problemMatcher": [
        "$haxe-absolute",
        "$haxe",
        "$haxe-error",
        "$haxe-trace"
    ],
    "group": {
        "kind": "build",
        "isDefault": true
    }
}
The task will be ignored.
```

**Comment by Gama11** [comment]
Hm... I've just tried reproducing this, and this _does_ generally work. However, during this I noticed that this error can happen when vshaxe hasn't been activated yet at the time you try to run the task (e.g. when you reload the window and don't have a `.hx` file open).

Having a `.hxml` file in the project's root directory happens to activate vshaxe via the `workspaceContains:*.hxml` activation event.
